app/views/bicycles/index.html.erb

```diff
@@ -1,54 +1,150 @@
-<p style="color: green"><%= notice %></p>
-<%= render 'shared/banner' %>
+<% if user_signed_in? %>
+  <div class="container d-flex flex-row justify-content-center align-items-stretch">
+    <div>
+      <div>
+        <form class="d-md-flex justify-content-center mt-4">
+          <div class="form-group mx-2">
+            <input type="text" class="form-control" placeholder="Où">
+          </div>
+          <div class="form-group mx-2">
+            <input type="text" class="form-control" placeholder="Quand">
+          </div>
+          <div class="form-group mx-2">
+            <select class="form-control" name="category">
+              <option value="" selected disabled hidden>Catégorie</option>
+              <option value="option1">Option 1</option>
+              <option value="option2">Option 2</option>
+              <option value="option3">Option 3</option>
+            </select>
+          </div>
+        </form>
+        <button type="submit" class="btn btn-primary mx-6">Rechercher</button>
+      </div>
+      <div id="bicycles" class="row d-flex justify-content-center">
+        <div id="carouselExampleControls" class="carousel slide col-md-7 mx-auto" data-bs-ride="carousel">
+          <div class="carousel-inner">
+            <% @bicycles.each_slice(4).with_index do |group, index| %>
+              <div class="carousel-item <%= index == 0 ? 'active' : '' %>">
+                <div class="row">
```

Comment:
`<div class="row justify-content-center">`

Reply:
@100PBaguuette Do you wish to proceed with this edit ? 💯 
Ready for approving those changes once done for a test in production asap. 👍